packages/frontend/src/pages/login.tsx

```diff
@@ -110,6 +110,7 @@ const Login: FC = () => {
             <Redirect
                 to={{
                     pathname: '/register',
+                    state: { from: location },
```

Comment:
We want to keep the state from the login redirect. Otherwise after register we always redirect to the login page.
```suggestion
                    state: { from: location.state.from },
```